models/facial_expression_recognition/demo.cpp

```diff
@@ -0,0 +1,300 @@
+        cv::Scalar(255,   0, 255), // right mouth corner
+        cv::Scalar(  0, 255, 255)  // left mouth corner
+    };
+    static cv::Scalar text_color{0, 255, 0};
+
+    auto output_image = image.clone();
+
+    if (fps >= 0)
+    {
+        cv::putText(output_image, cv::format("FPS: %.2f", fps), cv::Point(0, 15), cv::FONT_HERSHEY_SIMPLEX, 0.5, text_color, 2);
+    }
+
+    for (int i = 0; i < faces.rows; ++i)
+    {
+        // Draw bounding boxes
+        int x1 = static_cast<int>(faces.at<float>(i, 0));
+        int y1 = static_cast<int>(faces.at<float>(i, 1));
+        int w = static_cast<int>(faces.at<float>(i, 2));
+        int h = static_cast<int>(faces.at<float>(i, 3));
+        cv::rectangle(output_image, cv::Rect(x1, y1, w, h), box_color, 2);
+
+        // Expression as text
+        String exp = expressions[i];
+        cv::putText(output_image, exp, cv::Point(x1, y1+12), cv::FONT_HERSHEY_DUPLEX, 0.5, text_color);
+
+        // Draw landmarks
+        for (int j = 0; j < landmark_color.size(); ++j)
+        {
+            int x = static_cast<int>(faces.at<float>(i, 2*j+4)), y = static_cast<int>(faces.at<float>(i, 2*j+5));
+            cv::circle(output_image, cv::Point(x, y), 2, landmark_color[j], 2);
+        }
+    }
+    return output_image;
+}
+
+string keys =
+"{ help  h          |                                                                  | Print help message. }"
+"{ model m          | facial_expression_recognition_mobilefacenet_2022july.onnx        | Usage: Path to the model, defaults to facial_expression_recognition_mobilefacenet_2022july.onnx  }"
+"{ yunet_model ym   | ../face_detection_yunet/face_detection_yunet_2023mar.onnx        | Usage: Path to the face detection yunet model, defaults to face_detection_yunet_2023mar.onnx  }"
+"{ input i          |                                                                  | Path to input image or video file. Skip this argument to capture frames from a camera.}"
+"{ backend_target t | 0                                                                | Choose one of the backend-target pair to run this demo:\n"
+                                                                                        "0: (default) OpenCV implementation + CPU,\n"
+                                                                                        "1: CUDA + GPU (CUDA),\n"
+                                                                                        "2: CUDA + GPU (CUDA FP16),\n"
+                                                                                        "3: TIM-VX + NPU,\n"
+                                                                                        "4: CANN + NPU}"
+"{ save s           | false                                                             | Specify to save results.}"
+"{ vis v            | false                                                             | Specify to open a window for result visualization.}"
```

Comment:
I propose to set `vis` defaults to true for a straight impression in a first run.